src/components/ImgWrapper/ImgWrapper.types.ts

```diff
@@ -0,0 +1,18 @@
+export type ImgWrapperSize = 'small' | 'default' | 'big';
+export const imgWrapperSize = ['small', 'default', 'big'];
```

Comment:
It seems for other components (Button, Badges, Chip ...) we use the pluriel to describe the array `imgWrapperSizes`.
I don't know which convention we want to apply.

Reply:
I updated it to be the same as the other components. Nice catch 👀